CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## Unreleased
+
+**Breaking Changes**
+
+### Map Tokens [#1411](https://github.com/hashicorp/terraform-cdk/pull/1411)
+
+As part of an effort to use more native types, there are now tokens for maps of primitive values.
+
+As a result, there is a minor breaking change:
+
+- Map attributes have gone from `{ [key: string]: TYPE } | cdktf.IResolvable` to `{ [key: string]:TYPE }` when `TYPE` is `string, number, or boolean`.
+  - The most common impact is maps created by using Terraform functions (`Fn.(...)`) will now need to be passed to `Token.as<String/Number/Boolean>Map()` before assigning to a resource attribute.
```

Comment:
So the bottom line is adding `| cdktf.IResolvable` would make TF functions easier to use, but it would obfuscate the type in Java, C#, and Go right?

Reply:
I checked with @ansgarm and he too is okay with that API change :)